api/src/main/java/org/openmrs/module/idgen/SequentialIdentifierGenerator.java

```diff
@@ -31,6 +31,7 @@
 	private Integer minLength; // If > 0, will always return identifiers with a minimum of this length
 	private Integer maxLength; // If > 0, will always return identifiers no longer than this length
     private String baseCharacterSet; // Enables configuration in appropriate Base
+	private Integer nextSequenceValue; // Pre-generate the next identifier
 
```

Comment:
The way I have it working locally now is that I have marked the "nextSequenceValue" property as not updateable. When a POST is made to save the SequentialIdentifierGenerator with the nextSequenceValue updated, I make a call directly to the identifierSourceService.saveSequenceValue function.
 How does this sound? 

(I think this way is better because it addresses the concerned raised, as well as provides a full REST interface to the SequentialIdentifierGenerator resource.)